utils/text/levenshtein.py

```diff
@@ -0,0 +1,37 @@
+# Third Party (PyPI) Imports
+import numpy
+
+
+def getLevenshteinDistance(job_title, title_from_json):
```

Comment:
rename signature to be more generic:

`def levenshtein(w1, w2)`

Reply:
Done